Podverse/PVDeleter.swift

```diff
@@ -49,10 +55,9 @@ class PVDeleter: NSObject {
     static func deleteEpisode(episodeId: NSManagedObjectID, fileOnly: Bool = false, shouldCallNotificationMethod: Bool = false) {
         
         let pvMediaPlayer = PVMediaPlayer.shared
-        
+        let moc = CoreDataHelper.createMOCForThread(threadType: .privateThread)
```

Comment:
I see I was setting variables inside dispathQueue methods, but I should be setting vars prior to those methods.

Reply:
Oh crap, you are right.. this is a mistake. I'll revert